src/main/java/no/unit/nva/model/contexttypes/Degree.java

```diff
@@ -0,0 +1,71 @@
+package no.unit.nva.model.contexttypes;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import no.unit.nva.model.Level;
+
+import java.util.List;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Degree extends Book {
```

Comment:
What is a Degree?

Reply:
A thesis in practice, I do not know why this name has been chosen.